install/kubernetes/helm/istio/values-istio-all.yaml

```diff
@@ -15,6 +15,9 @@ grafana:
   enabled: true
 
 zipkin:
+  enabled: false
```

Comment:
should we disable one and keep the other? Like have Jaeger but not zipkin?

Reply:
If #5079 is definitely the approach that is wanted, then yes it may be worth just having the jaeger property - otherwise I could revert the change in that file and leave as it was.

Reply:
Given how we configure envoy - I don't think we can have both, user will need to chose one or the other, and the main requirement is to be able to have a smooth switch. 

Unfortunately we use the name 'zipkin' in the service - we could add a 'tracing' service but we should keep zipkin for backward compat (or we can keep using zipkin service, but have it point to the jaegar deployment ).